friendly/src/main/java/com/google/sps/servlets/GroupServlet.java

```diff
@@ -39,12 +40,35 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/group")
+public class GroupServlet extends HttpServlet {
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    UserService userService = UserServiceFactory.getUserService();
+
+    // user not logged in, do not have groups
+    if (!userService.isUserLoggedIn()){
+        response.sendError(403, "Not authorized to comment.");
+        return;
+    }
+    String ownerId = userService.getCurrentUser().getUserId();
+    
+
+	Query query = new Query("Group")
```

Comment:
I think there's an indentation typo here

Reply:
ok, done. thanks for pointing it out!